AirCasting/SDSync/ViewsFlow/BackendSyncCompleted/BackendSyncCompletedViewModel.swift

```diff
@@ -0,0 +1,40 @@
+// Created by Lunar on 02/12/2021.
+//
+
+import Foundation
+
+enum ProceedToSyncView {
+    case restart
+    case bluetooth
+}
+
+protocol BackendSyncCompletedViewModel: ObservableObject {
+    var presentRestartNextScreen: Bool { get set }
+    var presentBTNextScreen: Bool { get set }
```

Comment:
please make those as others we did today -> get only and use functions to inform viewmodel